warehouse/migrations/versions/c0c0544354e7_add_totp_secret_field.py

```diff
@@ -0,0 +1,58 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+add totp secret field
+
+Revision ID: c0c0544354e7
+Revises: c4a1ee483bb3
+Create Date: 2019-03-12 14:36:48.791870
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+revision = "c0c0544354e7"
+down_revision = "c4a1ee483bb3"
+
+# Note: It is VERY important to ensure that a migration does not lock for a
+#       long period of time and to ensure that each individual migration does
+#       not break compatibility with the *previous* version of the code base.
+#       This is because the migrations will be ran automatically as part of the
+#       deployment process, but while the previous version of the code is still
+#       up and running. Thus backwards incompatible changes must be broken up
+#       over multiple migrations inside of multiple pull requests in order to
+#       phase them in over multiple deploys.
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
```

Comment:
This migration doesn't seem like it matches the current state of the models at all, and we then immediately drop these columns in the next migration. Doesn't seem worthwhile to keep it in this PR?

Reply:
Yep, this is old. I'll remove it.